textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/join/JoinTest.java

```diff
@@ -1,2015 +0,0 @@
-package edu.uci.ics.textdb.dataflow.join;
-
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.stream.Collectors;
-
-import org.apache.lucene.analysis.Analyzer;
-import org.apache.lucene.analysis.standard.StandardAnalyzer;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
-
-import edu.uci.ics.textdb.api.common.Attribute;
-import edu.uci.ics.textdb.api.common.FieldType;
-import edu.uci.ics.textdb.api.common.IField;
-import edu.uci.ics.textdb.api.common.ITuple;
-import edu.uci.ics.textdb.api.common.Schema;
-import edu.uci.ics.textdb.api.dataflow.IOperator;
-import edu.uci.ics.textdb.api.storage.IDataStore;
-import edu.uci.ics.textdb.common.constants.DataConstants;
-import edu.uci.ics.textdb.common.constants.SchemaConstants;
-import edu.uci.ics.textdb.common.exception.DataFlowException;
-import edu.uci.ics.textdb.common.field.DataTuple;
-import edu.uci.ics.textdb.common.field.IntegerField;
-import edu.uci.ics.textdb.common.field.ListField;
-import edu.uci.ics.textdb.common.field.Span;
-import edu.uci.ics.textdb.common.field.StringField;
-import edu.uci.ics.textdb.common.field.TextField;
-import edu.uci.ics.textdb.common.utils.Utils;
-import edu.uci.ics.textdb.dataflow.common.FuzzyTokenPredicate;
-import edu.uci.ics.textdb.dataflow.common.IJoinPredicate;
-import edu.uci.ics.textdb.dataflow.common.JoinDistancePredicate;
-import edu.uci.ics.textdb.dataflow.common.KeywordPredicate;
-import edu.uci.ics.textdb.dataflow.fuzzytokenmatcher.FuzzyTokenMatcher;
-import edu.uci.ics.textdb.dataflow.keywordmatch.KeywordMatcherSourceOperator;
-import edu.uci.ics.textdb.dataflow.projection.ProjectionOperator;
-import edu.uci.ics.textdb.dataflow.projection.ProjectionPredicate;
-import edu.uci.ics.textdb.dataflow.source.IndexBasedSourceOperator;
-import edu.uci.ics.textdb.dataflow.utils.TestUtils;
-import edu.uci.ics.textdb.storage.DataStore;
-import edu.uci.ics.textdb.storage.writer.DataWriter;
-import junit.framework.Assert;
-
-/**
- * 
- * @author sripadks
- *
- */
-public class JoinTest {
-    private KeywordMatcherSourceOperator keywordSourceOuter;
-    private KeywordMatcherSourceOperator keywordSourceInner;
-    private DataWriter dataWriterForOuter;
-    private DataStore dataStoreForOuter;
-    private DataWriter dataWriterForInner;
-    private DataStore dataStoreForInner;
-    private Analyzer analyzer;
-    private Join join;
-    List<ITuple> bookTuple1;
-    List<ITuple> bookTuple2;
-    List<Attribute> attributeList;
-    List<Attribute> modifiedAttributeList;
-    private int maxVal = Integer.MAX_VALUE;
-
-    // This method sets up some stuff before beginning each test.
-    @Before
-    public void setup() throws Exception {
```

Comment:
The logic in `setUp()` (writing the join index files) is moved to `JoinDistanceHelper.java`.